modules/webhook/tests/test.bal

```diff
@@ -0,0 +1,254 @@
+// Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/config;
+import ballerina/test;
+import ballerina/websub;
+import ballerinax/twilio;
+import ballerina/io;
+import ballerina/runtime;
+import ballerina/log;
+
+listener Listener twilioListener = new (8080);
+
+boolean smsQueuedNotified = false;
+boolean smsSentNotified = false;
+boolean smsRecievedNotified = false;
+
+boolean callQueuedNotified = false;
+boolean callRingingNotified = false;
+boolean callInProgressNotified = false;
+boolean callCompletedNotified = false;
+
+
+// Mock service for testing webhook events
+
+@websub:SubscriberServiceConfig {subscribeOnStartUp: false}
+service websub:SubscriberService /twilio on twilioListener {
+    remote function onNotification(websub:Notification notification) {
+
+        io:println("");
+        io:println(notification.getFormParams().toBalString());
+        io:println("");
```

Comment:
Why we have system io in this code? Can't we use log interfaces?

Reply:
those were added for a debug purpose, and have been missed out to remove. now removed